src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/jdbc_intro";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "Nk102030!";

```

Comment:
Storing passwords in source code is a security risk. Consider using environment variables or a configuration file for sensitive data.